sdk/profiles/src/main/java/io/opentelemetry/sdk/profiles/data/AttributeUnitData.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.profiles.data;
+
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * Represents a mapping between Attribute Keys and Units.
+ *
+ * @see "pprofextended.proto::AttributeUnit"
+ */
+@Immutable
+public interface AttributeUnitData {
```

Comment:
Easy enough to add e.g. `String getUnit()` to the interface, but what's the expected implementation? Store the string in the bean, independent of any storage in the lookup table and thus potentially inconsistent, or hold a reference on the String[] table and index into it from the getter? Also, what's the expected use case? The marshaller won't call it, it doesn't care about the string value, only its index.

Reply:
Assuming you're ok living without this, now that the revised plan is to move these classes into the serialization package for use just by marshallers, rather than having them for general use in the sdk package.